src/attack.rs

```diff
@@ -169,3 +183,25 @@ fn breakable_system(
         }
     }
 }
+
+fn drop_system(
+    query: Query<(&Drop, &Transform, Entity), With<Breakable>>,
+    mut items_assets: ResMut<Assets<ItemMeta>>,
+    mut commands: Commands,
+) {
+    for (drop, transform, entity) in &query {
+        if drop.drop {
+            let ground_offset = Vec3::new(0.0, consts::GROUND_Y, consts::ITEM_LAYER);
+
+            let item_spawn_meta = ItemSpawnMeta {
+                location: transform.translation - ground_offset,
+                item: String::new(),
+                item_handle: items_assets.add(drop.item.clone()),
+            };
+            let item_commands = commands.spawn_bundle(ItemBundle::new(&item_spawn_meta));
+            ItemBundle::spawn(item_commands, &item_spawn_meta, &mut items_assets);
+
+            commands.entity(entity).despawn_recursive();
+        }
+    }
+}
```

Comment:
Maybe this could be moved to `item.rs` as well, but it would require either implementing plugin for `ItemPlugin`, or adding the system in main.rs.

Reply:
Imo I would prefer for breakable to throw an event and this system just listens to it

Reply:
As mentioned in my other comment, :+1: for using an event, but we still need the system, so I think it would make sense to add to the `ItemPlugin`.